planner/core/plan_cost_ver2.go

```diff
@@ -212,12 +206,8 @@ func (p *PhysicalTableReader) getPlanCostVer2(taskType property.TaskType, option
 
 // getPlanCostVer2 returns the plan-cost of this sub-plan, which is:
 // plan-cost = index-side-cost + (table-side-cost + double-read-cost) / double-read-concurrency
-// index-side-cost = (index-child-cost + index-net-cost + index-seek-cost) / dist-concurrency # same with IndexReader
-// table-side-cost = (table-child-cost + table-net-cost + table-seek-cost) / dist-concurrency # same with TableReader
-// double-read-cost = double-read-seek-cost + double-read-cpu-cost
-// double-read-seek-cost = double-read-tasks * seek-factor
-// double-read-cpu-cost = index-rows * cpu-factor
-// double-read-tasks = index-rows / batch-size * task-per-batch # task-per-batch is a magic number now
```

Comment:
Why do we remove the comments? It seems that we only need to remove `index-seek-cost` and `table-seek-cost`.

Reply:
I add them back, PTAL